.github/workflows/npm-publish.yml

```diff
@@ -8,26 +8,15 @@ on:
     types: [created]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: 14
-      - run: yarn install
-      - run: yarn build
-
   publish-npm:
-    needs: build
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
           node-version: 14
           registry-url: https://registry.npmjs.org/
-      - run: yarn install
+      - run: yarn install --frozen-lockfile
```

Comment:
Now build is missing.

```suggestion
      - run: yarn install --frozen-lockfile
      - run: yarn build
```

Reply:
Ya ya, I know, ignored as we already have `prepublishOnly` script which build before publishing https://github.com/snapshot-labs/snapshot.js/blob/master/package.json#L53

Reply:
Does it?
https://github.com/yarnpkg/berry/issues/2305
https://yarnpkg.com/advanced/lifecycle-scripts

(Didn't know that too... 🤫)

Reply:
Yea with yarn we may have to change it to `prepack + prepublish` that's why we are still using `npm publish` 

which work fine in our current workflow
https://github.com/snapshot-labs/snapshot.js/runs/5079073430?check_suite_focus=true
![image](https://user-images.githubusercontent.com/15967809/152737206-bc89bd75-1ff1-469f-8416-cc30e12b49bc.png)

Reply:
Ah ok cool! Got it.